### PR TITLE
Tech debt standards workflow document

### PR DESCRIPTION
**Pull request by msecret**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,72 @@ We're so glad you're thinking about contributing to an 18F open source project!
...
+- What is a good commit message?
+- What to look for in code review?
+- Should code be pulled down and tested for each review?
+- Any time you change a file, you should turn on linting for that file and fix the errors
+- Any new files should be linted.
+- Currently, only main cg-deck repo will work for CI tests, forks can't run it due to privacy issues.
+- How to share credentials required to run things?
+
+#### Current considerations
+- Working off a branch that gets deployed to a non-used site (no feedback on bugs/etc)
```
Comment (jcscottiii) [inline CONTRIBUTING.md]: looking into this https://github.com/pivotal-cf/pcfdev so we can test it against a pseudo-live env then creds can be dummy ones.

Comment (msecret) [inline CONTRIBUTING.md]: @jcscottiii that would be SO awesome

**Comment by jeremy-lyft** [comment]
:shipit:

**Comment by msecret** [comment]
@jcscottiii I mostly finished up this document. Let me know if there's anything you're not clear on or don't agree with

**Comment by jeremiak** [comment]
Looks good to me! Merging on down

**Pull request merged**